Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -243,20 +249,88 @@ In calculating the SCI value, you SHOULD use the highest granularity data availa
 - You SHOULD use a value for `R` from the specified [preset list](#preset-list-for-baselines) to compute `CI` but if you choose to use another value for `R`, you MUST provide a reason for that choice.
 
 
-## Boundaries 
+## Boundaries
 
 [placeholder]
 
 ## Exclusions
 
-Because this standard lays out a consequential methodology for calculating the emissions associated with a piece of software, the following must not be included in the calculation: 
+Because this standard lays out a consequential methodology for calculating the emissions associated with a piece of software, the following must not be included in the calculation:
 
-### Market-based Measures 
-**“Market-based measures"** are achieved through renewables matching (e.g. PPA for solar near a datacenter) that helps lower the grid carbon intensity for everyone in the area. From a carbon accounting perspective PPAs, RECs etc can allow datacenters to get to “100% renewable”, even if the grid is not 100% renewable 24x7 from a grid perspective. Market-based measures includes, but is not limited to the following: 
-- Carbon offsets 
-- Electricity Attribute Certificates (EACs) 
+### Market-based Measures
+**“Market-based measures"** are achieved through renewables matching (e.g. PPA for solar near a datacenter) that helps lower the grid carbon intensity for everyone in the area. From a carbon accounting perspective PPAs, RECs etc can allow datacenters to get to “100% renewable”, even if the grid is not 100% renewable 24x7 from a grid perspective. Market-based measures includes, but is not limited to the following:
+- Carbon offsets
+- Electricity Attribute Certificates (EACs)
 - Power Purchase Agreements (PPAs)
 - Renewable Energy Credits (RECs)
 
-### Infrastructure Measures 
+### Infrastructure Measures
 **“Infrastructure measures”** including any infrastructure that integrate renewables via a "direct wire connection" (e.g. a datacenter with solar panels on the roof and a battery storage located onsite). This is conceptually closer to a Microgrid, where there is a higher % of renewable energy usage than the local grid carbon intensity.
+
+## Reporting Requirements
+To comply with this specification and implement carbon transparency, an entity MUST meet these reporting requirements.
+
+### Data elements to be reported
+The following list contains the REQUIRED and OPTIONAL data elements to be reported. Some data elements are defined elsewhere in the specification as REQUIRED, and are included again for completeness.
+
+| Name | Identifier | Optionality | Format | Notes |
+| - | - | - | - | - |
+| Supporting infrastructure and systems contained within the [Software Boundary](#software-boundary) | sci/software-boundary | MUST | List | An array of names, or an array of key-value pairs.
```

Comment:
I'm not super confident Software Boundary can be explained with enough context as a bullet pointed list, was thinking initially as a free text field.